client/blocks/reader-share/index.jsx

```diff
@@ -97,6 +97,8 @@ class ReaderShare extends Component {
 			className: 'popover reader-share__popover',
 		};
 
+		const reblogTitle = this.props.comment && translate( "Turn this comment it's own post" );
```

Comment:
Missing a word I think:
```suggestion
		const reblogTitle = this.props.comment && translate( "Turn this comment into its own post" );
```

Reply:
ah yes, great catch!